dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowHistory.java

```diff
@@ -65,7 +69,30 @@ public void setCreationDate(Date creationDate) {
     }
 
     public String getChangeDescription() {
-        return changeDescription;
+
+        if (UtilMethods.isSet(this.changeDescription) && this.changeDescription.trim().startsWith("{")) {

```

Comment:
I think we have an `StringUtils.isJson` method.    Maybe it needs a null check and a trim() but those would be good to have.